python/python/glide/async_commands/core.py

```diff
@@ -2235,6 +2235,39 @@ async def zpopmax(
             ),
         )
 
+    async def bzpopmax(
+        self, keys: List[str], timeout: float
+    ) -> Optional[List[Union[str, float]]]:
+        """
+        Blocks the connection until it removes and returns a member with the highest score from the sorted sets stored
+        at the specified keys. The sorted sets are checked in the order they are provided.
```

Comment:
Note: the "keys" arg is not surrounded with angle quotes for other command docs in this file, so I've followed that pattern here